src/main/java/core/basesyntax/report/FruitBalance.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.report;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitBalance {
+    public static final Map<String, Integer> FRUIT_BALANCE = new HashMap<>();
+}
```

Comment:
do we need this class?

Reply:
It is used as storage for data, which is then used in creating reports. It is used by AdditionHandler, DecreaseHandler and ReportCreator, so I think that this class is needed.

Reply:
and what about `Database` class? should we have both of them?